libraries/hotstuff/include/eosio/hotstuff/chain_pacemaker.hpp

```diff
@@ -3,7 +3,7 @@
 #include <eosio/hotstuff/base_pacemaker.hpp>
 #include <eosio/hotstuff/qc_chain.hpp>
 
-#include <eosio/chain/finalizer_set.hpp>
+//#include <eosio/chain/finalizer_set.hpp>
```

Comment:
This should be removed. But `finalizer_set` needs to be forward declared.

Reply:
I did remove the commented out line, but I'm not sure if/how the forward declaration would be used, since finalizer_set is included in the base_pacemaker.hpp header.